src/GraphQL.ApiTests/ApiApprovalTests.cs

```diff
@@ -12,7 +11,7 @@ public class ApiApprovalTests
     {
         [Theory]
         [InlineData(typeof(IGraphType))]
-        [InlineData(typeof(DocumentWriter))]
+        [InlineData(typeof(IDocumentWriter))]
```

Comment:
Why? This is a type from the same assembly. There should be two types from two serializers assemblies.

Reply:
I guess I'm confused about what this test does. My thinking was it was for the public api we care about, but I guess it's any public methods in any packages. Will add both. Pretty sure this is the failing test too, but I don't understand the reason.

Reply:
This test helps you see changes in the public api (no changes/minor/breaking) and follow SemVer.

>  I guess it's any public methods in any packages

Yes, it should show public APIs of our three packages.

Reply:
It's still failing, but it's some kind of weird exception where it can't even exec the test. Will look after lunch but might need some help if you have a sec